customer-commons/src/test/java/no/unit/nva/customer/model/SecretManagerDoiAgentDaoTest.java

```diff
@@ -35,14 +35,34 @@ void shouldFailDeserialize() {
     }
 
     @Test
-    void shouldUpdate() {
-        var doiAgentDto = createSampleCustomerDao().toCustomerDto()
-                              .getDoiAgent().addPassword(randomString());
+    void shouldUpdateDoiAgent() {
+        var doiAgentDto =
+            createSampleCustomerDao()
+                .toCustomerDto()
+                .getDoiAgent()
+                .addPassword(randomString());
         var secretManagerDoiAgentDao = new SecretManagerDoiAgentDao(doiAgentDto);
```

Comment:
You create a SecretManagerDoiAgentDao from a CustomerDao. Then merge with the same CustomerDao instance. What is actually being tested here? Is it just for coverage?

Reply:
it is mostly for coverage.

I am merging two instances of  SecretManagerDoiAgentDao, one with an updated password.

Reply:
renamed tests and hopefully made it clearer.